_posts/2018-03-08-roadmap.md

```diff
@@ -0,0 +1,221 @@
+[custom test frameworks]: https://github.com/rust-lang/rfcs/pull/2318
+
+## Library improvements
+
+Building on [our work from last year][blitz], we will publish a 1.0 version of
+the [Rust API guidelines book], continue pushing important libraries to 1.0
+status, improve discoverability through a revamped cookbook effort, and make
+heavy investments in libraries in specific domains—as we’ll see below.
+
+[blitz]: https://blog.rust-lang.org/2017/05/05/libz-blitz.html
+[Rust API guidelines book]: https://github.com/rust-lang-nursery/api-guidelines
+
+**Web site improvements**
+
+As part of Rust 2018, we will completely overhaul the Rust web site, making it
+useful for CTOs and engineers alike. It should be far easier to find information
+to help evaluate Rust for your use case, and to stay up to date with the latest
+tooling and ecosystem improvements.
+
+## Four target domains
+
+Part of our goal with Rust 2018 is to demonstrate Rust’s productivity in
+specific domains of use. We’ve selected four such domains to invest in and
+highlight this year:
+
+- **Network services**. Rust’s reliability and low footprint make it an
+  excellent match for network services and infrastructure, especially at high
+  scale.
+- **CLI apps**. Rust’s portability, reliability, ergonomics, and ability to
+  produce static binaries come together to great effect for writing CLI apps.
+- **WebAssembly**. The “wasm” web standard allows shipping native-like binaries
+  to all major browsers, but GC support is still years away. Rust
+  is [extremely well positioned](https://mgattozzi.com/rust-wasm) to target this
+  domain, and provides a reasonable on-ramp for programmers coming from JS.
+- **Embedded devices**. Rust has the potential to make programming
+  resource-constrained devices much more productive—and fun! We want embedded
+  programming to reach first-class status this year.
+
+Each of these domains has a dedicated working group for the year. These WGs will
+work in a cross-cutting fashion, interfacing with language, tooling, library,
+and documentation work.
+
+## Better serving intermediate Rustaceans
+
+One of the strongest messages we’ve heard from production users, and [the 2017
+survey], is that people need more resources to take them from understanding
+Rust’s concepts to knowing how to use them _effectively_. The roadmap does not
+stipulate exactly what these resources should look like
+— [probably there should be several kinds][intermediate] — but commits us as a
+community to putting significant work into this space, and ending the year with
+some solid new material.
+
+[the 2017 survey]: https://blog.rust-lang.org/2017/09/05/Rust-2017-Survey-Results.html
+[intermediate]: https://quietmisdreavus.net/code/2018/01/10/not-a-layer-cake-analogy/
+
+## Community
+
+**Connect and empower Rust's global community**. We will pursue
+internationalization (specifically: translation) as a first-class concern, and
```

Comment:
"internationalization (specifically: translation)" is not very clear to me - do you mean adding internationalisation support to the language to facilitate translation or internationalising things like compiler messages and documentation by translating them?